src/LiveComponent/assets/src/ComponentRegistry.ts

```diff
@@ -2,22 +2,22 @@ import Component from './Component';
 import { getElementAsTagText } from './dom_utils';
 
 const ComponentRegistry = class {
-    private components = new WeakMap<HTMLElement, Component>();
+    #components = new WeakMap<HTMLElement, Component>();
```

Comment:
private should be ok - it’s a typescript thing (whereas # is pure js, so that’s also valid, but private should be ok in ts). Did this seem to fix a warning?